cinema_service/settings.py

```diff
@@ -25,7 +27,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get("DJANGO_DEBUG", "") != "False"
 
 ALLOWED_HOSTS = []
```

Comment:
The `ALLOWED_HOSTS` list is empty. Ensure to configure this setting appropriately for your deployment environment to prevent security vulnerabilities.